avocado/utils/service.py

```diff
@@ -161,22 +161,22 @@ def method(cmd_result):
                     service_name = sublines[0]
                     # Store the status of each target in status_per_target_dict
                     for target in range(7):
-                        status = sublines[target + 1].split(":")[-1]
+                        status = sublines[target + 1].split(b":")[-1]
```

Comment:
@lolyu I think we can use only strings instead of bytes here. You can change line 156 above to get `stdout_text`[1] instead of `stdout` from `cmd_result`. So we can keep using only string objects itself.

[1] - https://github.com/avocado-framework/avocado/blob/master/avocado/utils/process.py#L365